play-2.4/swagger-play2/app/play/modules/swagger/SwaggerPlugin.scala

```diff
@@ -81,10 +82,10 @@ class SwaggerPluginImpl @Inject()(lifecycle: ApplicationLifecycle, router: Route
   }
 
   val routesFile = config.underlying.hasPath("play.http.router") match {
-    case false => "routes"
+    case false => "conf/routes"
```

Comment:
I wonder if you need to prepend `conf/` to all of these? Maybe you could instead do `app.classloader.getResource(routesFile).getFile()`?